tarantool/connection_pool.py

```diff
@@ -1025,3 +1025,638 @@ def execute(self, query, params=None, *, mode=None):
+
+        :param space_name: Refer to
+            :paramref:`~tarantool.Connection.crud_replace_object.params.space_name`.
+
+        :param values: Refer to
+            :paramref:`~tarantool.Connection.crud_replace_object.params.values`.
+
+        :param opts: Refer to
+            :paramref:`~tarantool.Connection.crud_replace_object.params.opts`.
+
+        :param mode: Request mode.
+        :type mode: :class:`~tarantool.Mode`, optional
+
+        :rtype: :class:`~tarantool.crud.CrudResult`
+
+        :raise: :exc:`~tarantool.error.CrudModuleError`,
+            :exc:`~tarantool.error.DatabaseError`
+        """
+
+        return self._send(mode, 'crud_replace_object', space_name, values, opts)
+
+    def crud_replace_many(self, space_name, values, opts={}, *, mode=Mode.RW):
+        """
+        Execute an CRUD_REPLACE_MANY request on the pool server: 
+        replaces batch rows through the 
+        `crud <https://github.com/tarantool/crud#replace-many>`__.
+        Refer to :meth:`~tarantool.Connection.crud_replace_many`.
+
+        :param space_name: Refer to
+            :paramref:`~tarantool.Connection.crud_replace_many.params.space_name`.
+
+        :param values: Refer to
+            :paramref:`~tarantool.Connection.crud_replace_many.params.values`.
+
+        :param opts: Refer to
+            :paramref:`~tarantool.Connection.crud_replace_many.params.opts`.
+
+        :param mode: Request mode.
+        :type mode: :class:`~tarantool.Mode`, optional
+
+        :rtype: :class:`~tarantool.crud.CrudResult`
+
+        :raise: :exc:`~tarantool.error.CrudModuleError`,
+            :exc:`~tarantool.error.DatabaseError`
+        """
+
+        return self._send(mode, 'crud_replace_many', space_name, values, opts)
+
+    def crud_replace_object_many(self, space_name, values, opts={}, *, mode=Mode.RW):
+        """
+        Execute an CRUD_REPLACE_OBJECT_MANY request on the pool server: 
+        replaces batch object rows through the 
+        `crud <https://github.com/tarantool/crud#replace-many>`__.
+        Refer to :meth:`~tarantool.Connection.crud_replace_object_many`.
+
+        :param space_name: Refer to
+            :paramref:`~tarantool.Connection.crud_replace_object_many.params.space_name`.
+
+        :param values: Refer to
+            :paramref:`~tarantool.Connection.crud_replace_object_many.params.values`.
+
+        :param opts: Refer to
+            :paramref:`~tarantool.Connection.crud_replace_object_many.params.opts`.
+
+        :param mode: Request mode.
+        :type mode: :class:`~tarantool.Mode`, optional
+
+        :rtype: :class:`~tarantool.crud.CrudResult`
+
+        :raise: :exc:`~tarantool.error.CrudModuleError`,
+            :exc:`~tarantool.error.DatabaseError`
+        """
+
+        return self._send(mode, 'crud_replace_object_many', space_name, values, opts)
+
+    def crud_upsert(self, space_name, values, operations=[], opts={}, *, mode=Mode.RW):
```

Comment:
Since `crud_*` always deal with routers and `crud.router`s doesn't write anything on their own, `Mode.ANY` should be better here (and in any other call).

Reply:
Done